index.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable no-unused-vars */
+let books = [];
+const title = document.querySelector('.boot-title');
+const author = document.querySelector('.author');
+const btn = document.querySelector('.btn');
+const bookHolder = document.querySelector('#bookList');
+
+function addBook() {
```

Comment:
- [ ] I am unable to add a book because of the following errors: 
![Screenshot from 2021-10-18 22-56-51](https://user-images.githubusercontent.com/62705947/137805608-3ec781eb-811b-475f-b12c-116aad36e542.png)